docs/conf.py

```diff
@@ -15,7 +17,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "rapids-cmake"
-copyright = "2021, NVIDIA"
+copyright = "2019-2023, NVIDIA"
```

Comment:
The initial commit in this project dates to 2021.
```suggestion
copyright = "2021-2023, NVIDIA"
```